manila/share/manager.py

```diff
@@ -2135,7 +2135,7 @@ def create_share_instance(self, context, share_instance_id,
                 with excutils.save_and_reraise_exception():
                     error = ("Creation of share instance %s failed: "
                              "failed to allocate network")
-                    LOG.warning(error, share_instance_id)
+                    LOG.error(error, share_instance_id)
```

Comment:
Port quota exceeded is a client error, this should stay a warning in my opinion.
We even provide a nice user message.

Reply:
it's client error, I agree. My concern is that the error logging are not consistent in this method. There are port limit error, security service error and no share server error. And below there is CIFS error. They are all handled similarly but slightly different. I would say, either log them all on warning or log them all on error. but not with different levels.